src/routes/curator-actions/batch/everything/+page.svelte

```diff
@@ -101,180 +105,336 @@
+						</p>
+					</div>
+				</div>
+			</form>
+		{/if}
+	</div>
 </div>
 
 <style>
+	.containerWrapper {
+		display: flex;
+		justify-content: center;
+		overflow-y: hidden;
+	}
+	.container {
+		margin-top: 13px;
+		background-color: theme('colors.backgroundBounty');
+		border-radius: 10px 10px 0px 0px;
+		width: 754px;
+	}
+	.header {
+		display: flex;
+		flex-direction: column;
+		gap: 7px;
+		margin: 18px 12px;
+	}
+	.bountyTitle {
+		font-size: 20px;
+		word-break: break-word;
+		font-weight: 300;
+	}
+	.bountyName {
+		font-weight: 800;
+	}
+	.text {
+		font-size: 12px;
+	}
+	.form {
+		display: flex;
+		flex-direction: column;
+		padding: 0px 5px 5px;
+	}
+	.headerForm {
+		font-size: 18px;
+		font-weight: 800;
+		background-color: theme('colors.backgroundButtonLight');
+		padding: 9px 7px;
+	}
+	.contentForm {
+		background-color: theme('colors.backgroundApp');
+		padding: 6px 8px;
+	}
+	.stepsList {
+		display: flex;
+		gap: 60px;
+	}
+	.textContent {
+		font-size: 14px;
+		list-style-type: decimal;
+		list-style-position: inside;
+	}
 	.cardsGrid {
-		grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
-		grid-auto-rows: 1fr;
+		display: grid;
+		grid-template-columns: repeat(2, 1fr);
+		gap: 24px;
+		padding: 10px 0px 30px;
+	}
+	.card {
+		background-color: theme('colors.backgroundBounty');
+		padding: 15px 10px;
+		border-radius: 2px;
+		border: 1px solid theme('colors.backgroundButtonDark');
+		position: relative;
+		min-height: 452px;
+	}
+	.blurred {
+		filter: blur(3px);
+		pointer-events: none;
+		position: absolute;
+	}
+	.balance,
+	.cardIndex {
+		font-weight: 800;
+	}
+
+	.index {
+		font-size: 24px;
+	}
+	.inputFields {
+		display: flex;
+		flex-direction: column;
+		gap: 18px;
+	}
+	.deleteButton {
+		opacity: 0.4;
+		align-self: self-start;
+	}
+	.overlay {
+		position: absolute;
+		left: 50%;
+		transform: translate(-50%, 200%);
+		align-self: center;
+		display: flex;
+		flex-direction: column;
+		font-size: 12px;
+		align-items: center;
+		justify-self: center;
+		gap: 5px;
+	}
+	.addButton {
+		background-color: theme('colors.backgroundButtonLight');
+		border-radius: 50%;
+		width: 40px;
+		height: 40px;
+		font-size: 30px;
+		font-weight: 200;
+	}
+	.restContainer {
+		width: 360px;
+		padding: 30px 7px;
+		display: flex;
+		flex-direction: column;
+		justify-self: center;
+		gap: 30px;
+	}
+	.extendBounty,
+	.claimChildBounty {
+		display: flex;
+		justify-content: space-between;
+		width: 100%;
+		align-items: center;
+	}
+	.signButton {
+		width: 100%;
+		height: 40px;
+		border-radius: 10px;
+		color: white;
+		background-color: theme('colors.accent');
+		transition:
+			background-color 0.3s ease,
+			transform 0.2s ease;
+	}
+	.signButton:hover {
+		transform: scale(1.02);
+	}
+	.signButton:disabled {
+		cursor: not-allowed;
+		opacity: 30%;
+	}
+	@media (max-width: 640px) {
+		.container {
+			width: 100%;
+		}
+		.cardsGrid {
+			grid-template-columns: repeat(1, 1fr);
```

Comment:
There are two ways to group rules. One is to group by the media query and another by the class name. The current approach has an upside of seeing at once everything that changes on a different screen size. Personally, I believe it is more helpful to see at once everything related to a particular element. Like "here’s the default value of grid-template-columns and here’s the override for big screen". I’m just sharing my experience here, feel free to keep the current approach if you like.